man/truelength.Rd

```diff
@@ -54,5 +54,22 @@ length(DT)                 # 2 used
 truelength(DT)             # 2050 allocated, 2048 free
 DT[,c:=7L]                 # add new column by assigning to spare slot
 truelength(DT)-length(DT)  # 2047 slots spare
+
+# FAQ: Dealing with data.table loaded from RDS or RData files
```

Comment:
This example doesn't use any of the functions documented in this .Rd page. I don't think it belongs here. A link to `setDT()` is enough.